internal/file_system.go

```diff
@@ -11,8 +11,11 @@ import (
 	"path"
 )
 
-func ProcessCredentialsTemplate(credentials *sso.GetRoleCredentialsOutput) string {
-	template := `[default]
+func ProcessCredentialsTemplate(credentials *sso.GetRoleCredentialsOutput, profile string) string {
+	if len(profile) == 0 {
```

Comment:
We could initialize the flag with a value `default` - so this check would be superfluous as profile would be always default or the custom value